hack/make/.integration-test-helpers

```diff
@@ -161,22 +161,6 @@ test_env() {
 	)
 }
 
-error_on_leaked_containerd_shims() {
-	if [ "$(go env GOOS)" = 'windows' ]; then
-		return
-	fi
-
-	leftovers=$(ps -ax -o pid,cmd \
```

Comment:
curious; could these leaks also happen with the v2 shims? If so, would it be useful to adjust this check instead to check for leaked v2 shims?

Reply:
Leaks may happen, but I'm not sure how we can port this script to the v2 shim, as the arguments are different


### `ps -ax -o pid,cmd`
v1:
```
 8956 containerd-shim -namespace moby -workdir /var/lib/containerd/io.containerd.runtime.v1.linux/moby/1db4006f0d10fb56bdc0e0055002f16d1da2d7df12440aa764f48f24dcd663fd -address /run/containerd/containerd.sock -containerd-binary /usr/bin/containerd -runtime-root /var/run/docker/runtime-runc
```

v2:
```
 9049 /usr/bin/containerd-shim-runc-v2 -namespace moby -id 23a1bcc8ce6724e8383189a5ca72bab49eb3bf423fbc0f08f632ab6a37a54b07 -address /run/containerd/containerd.sock
```

Reply:
And probably the script had been broken even for the v1 shim. `$4` is typically like "moby", and won't hit the `/.*\/bundles\/.*\/test-integration/` regexp.

Reply:
Yeah, not a major issue; I was mostly wondering; apparently we added this check at some point because we ran into issues, so I was wondering if we should have a check if those issues still exist.

Not a blocker for this PR 👍